FluentUI.Demo/src/main/java/com/microsoft/fluentuidemo/demos/V2ActionBarActivity.kt

```diff
@@ -0,0 +1,193 @@
+                            selected = (selectedActionBarPos == actionBarPos[1]),
+                            onClick = {
+                                selectedActionBarPos = actionBarPos[1]
+                            }
+                        )
+                    }
+
+                }
+                ListItem.Header(title = resources.getString(R.string.actionbar_type_heading))
+                Column(
+                    modifier = Modifier.padding(16.dp),
+                    verticalArrangement = Arrangement.spacedBy(10.dp),
+                ) {
+                    Row(
+                        horizontalArrangement = Arrangement.spacedBy(16.dp),
+                        verticalAlignment = Alignment.CenterVertically,
+                        modifier = Modifier.fillMaxWidth()
+                    ) {
+                        BasicText(
+                            text = resources.getString(R.string.actionbar_basic_radio_label),
+                            modifier = Modifier.weight(1F),
+                            style = TextStyle(
+                                color = FluentTheme.aliasTokens.neutralForegroundColor[FluentAliasTokens.NeutralForegroundColorTokens.Foreground3].value(
+                                    themeMode = ThemeMode.Auto
+                                )
+                            )
+                        )
+                        RadioButton(
+                            modifier = Modifier.testTag(TAB_BAR_VERTICAL_RADIO),
+                            selected = (selectedActionBarType == actionBarType[0]),
+                            onClick = {
+                                selectedActionBarType = actionBarType[0]
+                            }
+                        )
+                    }
+                    Row(
+                        horizontalArrangement = Arrangement.spacedBy(16.dp),
+                        verticalAlignment = Alignment.CenterVertically,
+                        modifier = Modifier.fillMaxWidth()
+                    ) {
+                        BasicText(
+                            text = resources.getString(R.string.actionbar_icon_radio_label),
+                            modifier = Modifier.weight(1F),
+                            style = TextStyle(
+                                color = FluentTheme.aliasTokens.neutralForegroundColor[FluentAliasTokens.NeutralForegroundColorTokens.Foreground3].value(
+                                    themeMode = ThemeMode.Auto
+                                )
+                            )
+                        )
+                        RadioButton(
+                            modifier = Modifier.testTag(TAB_BAR_HORIZONTAL_RADIO),
+                            selected = (selectedActionBarType == actionBarType[1]),
+                            onClick = {
+                                selectedActionBarType = actionBarType[1]
+                            }
+                        )
+                    }
```

Comment:
Since most of these rows are same with only a difference in arguments, can we put them in private composable a function and just pass the different arguments